src/components/Book.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({
+  id, title, category,
+}) => (
```

Comment:
- [ ] The Book component should accept the `book` object as a prop.
 * You don't have to break down the book properties at the prop level. You can just pass the book object, which will be a representation of the book attributes (id, title, category)

Reply:
👍